docs/source/Support/bskReleaseNotes.rst

```diff
@@ -76,6 +76,7 @@ Version |release|
 - Fixed use of spherical coordinate system in :ref:`magneticFieldWMM` model.
 - Added ability to run the GitHub ``pull_request.yml`` action on a select branch
 - Fixed mass depletion rate bug in :ref:`thrusterStateEffector` previously fixed at 100%
+- Enhanced :ref:`thrusterDynamicEffector` to allow automatic scaling down of thrust and Isp as fuel mass depletes.
```

Comment:
after rebasing to latest develop, please add the release notes line to the current 2.4.x beta cycle section of the release notes.

Reply:
I've moved this to the current beta cycle section now.